src/System.Management.Automation/FormatAndOutput/common/PSStyle.cs

```diff
@@ -572,7 +572,7 @@ public FileInfoFormatting()
         /// <summary>
         /// Gets or sets the rendering mode for output.
         /// </summary>
-        public OutputRendering OutputRendering { get; set; } = OutputRendering.Ansi;
+        public OutputRendering OutputRendering { get; set; } = OutputRendering.Host;
```

Comment:
Why and where it was changed to Ansi? :-)

Reply:
This was a temporary change to address https://github.com/PowerShell/Announcements/issues/28.  This PR is trying to fix the underlying problem so we can change it back.